docs/Automation.md

```diff
@@ -188,78 +210,41 @@ Sample `C#` code—this is interactive, but yours doesn’t need to be:
             /// </summary>
             static void Main(string[] args)
             {
-                var parameters = new Dictionary<string, string>();
-                string secondaryConfigFile = string.Empty;
-                char[] delimiters = {'='};
-                foreach (string arg in args)
+                string testAppPath = Path.GetFullPath(@"..\myApplication.exe");
+                string outputDir = Path.GetFullPath(@".\TestOutput");
+                Process testProcess = Process.Start(testAppPath);
+                var config = Config.Builder.ForProcessId(testProcess.Id)
+                    .WithOutputFileFormat(OutputFileFormat.A11yTest)
+                    .WithOutputDirectory(outputDir)
+                    .Build();
+
+                var scanner = ScannerFactory.CreateScanner(config);
+
+                try
                 {
-                    string[] pieces = arg.Split(delimiters);
-                    if (pieces.Length == 2)
-                    {
-                        string key = pieces[0].Trim();
-                        string value = pieces[1].Trim();
-                        if (!string.IsNullOrWhiteSpace(key) && !string.IsNullOrWhiteSpace(value))
-                        {
-                            // Special case for SecondaryConfigFile
-                            if (key.Equals("SecondaryConfigFile", StringComparison.OrdinalIgnoreCase))
-                            {
-                                secondaryConfigFile = value;
-                            }
-                            else
-                            {
-                                parameters[key] = value;
-                            }
-                            continue;
-                        }
-                    }
-                    Console.WriteLine("Ignoring malformed input: {0}", arg);
-                };
-                Console.WriteLine(StartCommand.Execute(parameters secondaryConfigFile).ToString());
-                int autoFileId = 0;
-                while (true)
+                    var output = scanner.Scan();
+                    Assert.IsTrue(output.ErrorCount == 0);
```

Comment:
`Assert.AreEqual(0, output.ErrorCount)` might be a bit nicer. That way the log will tell us what the error count was if it fails
```suggestion
                    Assert.AreEqual(0, output.ErrorCount);
```